src/lib/viewers/image/ImageLoader.js

```diff
@@ -24,7 +24,7 @@ const VIEWERS = [
         NAME: 'Image',
         CONSTRUCTOR: ImageViewer,
         REP: 'jpg',
-        EXT: ['jpeg', 'jpg'],
+        EXT: ['jpeg', 'jpg', 'cr2', 'crw', 'dng', 'nef', 'raf', 'raw'],
```

Comment:
@qliu-box, to confirm: These extensions will be using JPG representations, not PNG. Is that correct?

Reply:
Yes, correct. Generating PNG for these raw images would take a long time so we decide to go with jpg